src/coreclr/src/vm/stubgen.h

```diff
@@ -261,6 +261,7 @@ class FunctionSigBuilder : protected StubSigBuilder
     }
 
 protected:
+    DAC_ALIGNAS(StubSigBuilder) // Align the first member to the alignment of the base class
```

Comment:
This type is not DACized. This should not be needed. Would it be better to ifdef it out for DAC?